user_docs/en/changes.t2t

```diff
@@ -12,19 +12,20 @@ What's New in NVDA
 - Initial support for Windows 10 on ARM64. (#7508)
 - Early support for reading and interactive navigation of mathematical content for Kindle books with accessible math. (#7536)
 - Added support for Azardi e-book reader. (#5848)
-- Version information for addons is now reported when being updated. (#5324)
-- Added support for new command line parameters to create a portable copy of NVDA. (#6329)
-- Support for Microsoft Edge running within Windows Defender Application Guard (#7600)
+- Version information for add-ons is now reported when being updated. (#5324)
+- Added new command line parameters to create a portable copy of NVDA. (#6329)
+- Support for Microsoft Edge running within Windows Defender Application Guard in Windows 10 Fall Creators Update. (#7600)
 - If running on a laptop or tablet, NVDA will now report when a charger is connected/disconnected, and when the screen orientation changes. (#4574, #4612)
+- New language: Macedonian.
 
 
 == Changes ==
-- The minimum supported Operating System for NVDA is now Windows 7 with Service Pack 1, or Windows Server 2008 R2 with Service Pack 1. (#7546)
+- The minimum supported Operating System for NVDA is now Windows 7 Service Pack 1, or Windows Server 2008 R2 Service Pack 1. (#7546)
 - Web dialogs in Firefox and Chrome web browsers now automatically use browse Mode, unless inside of a web application. (#4493)
 - In browse mode, tabbing and moving with quick navigation commands no longer announces jumping out of containers such as lists and tables, which makes navigating more efficient. (#2591)
 - In Browse mode for Firefox and Chrome, the name of form field groups are now announced when moving into them with quick navigation or when tabbing. (#3321)
 - In browse mode, the quick navigation command for embedded objects (o and shift+o) now includes audio and video elements as well as elements with the aria roles application and dialog. (#7239)
-- Espeak-ng has been updated (to commit 01919cd48a566cdf34347784b2e74554b376e900), this resolves some issues with producing release builds. (#7385)
+- Espeak-ng has been updated to commit 01919cd4, which resolves some issues with producing release builds. (#7385)

```

Comment:
Arguably, () around the commit stuff was more correct since it lets the non-essential info be in parenthesis. However, the way you did this is more compact. If noone else objects, I think this is better. Maybe a comma before to could make it slightly more obvious the commit info is extra, but it really doesn't matter.